evenodd.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+int main()
+{
+	int z;
+	double y, Your_Number, xx;
```

Comment:
Not the way I expected it to be implemented, but this is 100% valid and is actually better because it properly handles non-integer values. So awesome job! 💯